### PR TITLE
Remove outdated and incorrect install guide in favor of maintained guide

### LINKED ISSUE #207: Homebrew installation instructions are incorrect in Readme
States:
ruby -e "$(curl -fsSL https://raw.github.com/mxcl/homebrew/go)"

Should be:
ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"

### PR DESCRIPTION
**Pull request by bdunne**
fixes #207

### CODE REVIEW

**Comment by brandondunne** [comment]
@johnmark Please review.

cc @Fryguy

File: source/community/index.html.haml [hunk 1]
```diff
@@ -31,14 +31,14 @@ title: Community
...
-      
+
   %section.col-md-4
     :markdown
       ### Develop
 
       * [Developer Documentation](/documentation/development/)
       * [Upgrade an image in-place](update-image)
-      * [Install and deploy, from source](install-from-source)
+      * [Install and deploy, from source](https://github.com/ManageIQ/guides/blob/master/developer_setup.md)
```
Comment (Fryguy) [inline source/community/index.html.haml]: guides is already mirrored to the website, so probably better to point to the website link @ /documentation/development/developer_setup

**Comment by jrafanie** [comment]
@Fryguy Can you merge?

**Comment by brandondunne** [comment]
@chessbyte Please review

**Pull request merged**